src/components/Patient/PatientIndex.tsx

```diff
@@ -210,77 +229,72 @@ export default function PatientIndex({ facilityId }: { facilityId: string }) {
             </p>
           </div>
 
-          <div>
-            <div className="space-y-6">
-              <SearchInput
-                data-cy="patient-search"
-                options={searchOptions}
-                onSearch={handleSearch}
-                className="w-full"
-                autoFocus
-              />
+          <div className="space-y-6">
+            <SearchInput
+              data-cy="patient-search"
+              options={searchOptions}
+              onSearch={handleSearch}
+              className="w-full"
+              autoFocus
+            />
 
-              <div className="min-h-[200px]" id="patient-search-results">
-                {(!!phoneNumber ||
-                  (!!identifierSearch.config && !!identifierSearch.value)) && (
-                  <>
-                    {isFetching || !patientList ? (
-                      <div className="flex items-center justify-center h-[200px]">
-                        <Loading />
-                      </div>
-                    ) : !patientList.results.length ? (
-                      <div>
-                        <div className="flex flex-col items-center justify-center py-10 text-center">
-                          <h3 className="text-lg font-semibold">
-                            {t("no_patient_record_found")}
-                          </h3>
-                          <p className="text-sm text-gray-500 mb-6">
-                            {t("no_patient_record_text")}
-                          </p>
-                          <AddPatientButton outline />
-                        </div>
-                      </div>
-                    ) : (
-                      <div className="rounded-lg border border-gray-200">
-                        <Table>
-                          <TableHeader>
-                            <TableRow>
-                              <TableHead className="w-[300px]">
-                                {t("patient_name")}
-                              </TableHead>
-                              <TableHead>{t("phone_number")}</TableHead>
-                              <TableHead>{t("gender")}</TableHead>
+            <div className="min-h-[200px]" id="patient-search-results">
+              {(!!phoneNumber || isIdentifierValid) && (
```

Comment:
Revert the check here so we show the record not found even if regex isn't valid.

Instead, highlight the input box w/ a red outline (check phone number field's behavior).